action.yml

```diff
@@ -10,6 +10,9 @@ inputs:
   platform-version:
     description: 'Version of the programming platform used to build the web app; if no value is provided, Oryx will determine the version needed to build the repository.'
     required: false
+  output-directory:
+    description: 'The directory where the build output has to be put.'
+    required: false
 branding:
```

Comment:
Maybe combine these changes in action.yml into the other PR? We can use single PR for this.